src/components/Button/Button.js

```diff
@@ -0,0 +1,14 @@
+import './Button.css';
+
+function Button({type, text, action}) {
+  return (
+    <button
+      className="button"
+      type={type}
+      onClick={action}
+    > {text}
+    </button>
+  )
+}
```

Comment:
Dit component is eigenlijk overbodig. Als je maar één componentje terugstuurd wat ongeveer evenveel attribuut waardes heeft als het component eromheen dan vind ik het onnodig om er een wrapper component omheen te zetten